pkg/syncer/volume_health.go

```diff
@@ -116,22 +117,22 @@ func updateVolumeHealthStatus(ctx context.Context, k8sclient clientset.Interface
 	_, foundAnnHealthTS := pvc.Annotations[annVolumeHealthTS]
 	if !found || val != volHealthStatus || !foundAnnHealthTS {
 		// VolumeHealth annotation on pvc is changed, set it to new value
-		log.Debugf("updateVolumeHealthStatus: update volume health annotation for pvc %s/%s from old value %s to new value %s",
+		log.Infof("updateVolumeHealthStatus: update volume health annotation for pvc %s/%s from old value %s to new value %s",
 			pvc.Namespace, pvc.Name, val, volHealthStatus)
 		metav1.SetMetaDataAnnotation(&pvc.ObjectMeta, annVolumeHealth, volHealthStatus)
 		timeNow := time.Now().Format(time.UnixDate)
 		metav1.SetMetaDataAnnotation(&pvc.ObjectMeta, annVolumeHealthTS, timeNow)
-		log.Debugf("updateVolumeHealthStatus: set annotation for health to %s at time %s for pvc %s/%s", volHealthStatus, timeNow, pvc.Namespace, pvc.Name)
+		log.Infof("updateVolumeHealthStatus: set annotation for health to %s at time %s for pvc %s/%s", volHealthStatus, timeNow, pvc.Namespace, pvc.Name)
```

Comment:
Can we have a single log for log at line 125 and 120? And can you explain why is this log critical? Wouldn't the annotation change appear in the API server log as well?